### PR TITLE
Support ZOLOTO_LEGACY_AXIS

### PR DESCRIPTION
**Pull request by WillB97**
Does the job

### CODE REVIEW

File: sr/robot3/robot.py [hunk 1]
```diff
@@ -59,6 +60,8 @@ def __init__(
             Robot under. See :ref:`Environments` for more information.
         :param ignored_ruggeduinos: List of Ruggeduino serial numbers to ignore.
             See :ref:`Custom Ruggeduino Firmware` for more information.
+        :param legacy_camera_axis: Use the coordinate systems the camera used
+            prior to SR OS 2023.2.0.
```
Comment (trickeydan) [inline sr/robot3/robot.py]:
```suggestion
            prior to SR OS 2023.1.0.
```

The version we're releasing this week is 2023.1.0. Unless you meant the next version afterwards?

File: sr/robot3/robot.py [hunk 2]
```diff
@@ -85,6 +88,8 @@ def __init__(
         self._init_metadata()
 
         self._init_cameras(self.metadata.marker_offset)
+        if legacy_camera_axis:
```
Comment (trickeydan) [inline sr/robot3/robot.py:91]: Might be worth adding a long line here?

**Review by trickeydan — APPROVED** [review]
Technical function is good. Please update comment and consider asking a long line before merging

File: sr/robot3/robot.py (hunk 2 above)
Comment (raccube) [reply sr/robot3/robot.py:91]: Did you mean _log line_?

Comment (WillB97) [reply sr/robot3/robot.py:91]: Maybe 6fb79ce

Comment (trickeydan) [reply sr/robot3/robot.py:91]: > Did you mean log line?

I did.

Comment (WillB97) [reply sr/robot3/robot.py:91]: @trickeydan Was the log line I added good?

Comment (trickeydan) [reply sr/robot3/robot.py:91]: Yep.

**Pull request merged**